jgrapht-core/src/main/java/org/jgrapht/alg/shortestpath/BidirectionalAStarShortestPath.java

```diff
@@ -0,0 +1,373 @@
+/*
+ * (C) Copyright 2019-2019, by Semen Chudakov, Dimitrios Michail, Joris Kinable, Jon Robison, Thomas Breitbart and Contributors.
```

Comment:
Keep only the first name here. 

In general the following rule applied. In the header we only list the main author (some exceptions are probably in the codebase) and we list additional authors in the main class javadocs.